packages/cozy-harvest-lib/src/components/cards/WebsiteLinkCard.jsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import logger from '../../logger'
+import Card from 'cozy-ui/transpiled/react/Card'
+import { SubTitle, Caption } from 'cozy-ui/transpiled/react/Text'
+import Divider from 'cozy-ui/transpiled/react/MuiCozyTheme/Divider'
+import { ButtonLink } from 'cozy-ui/transpiled/react/Button'
+import { Media, Img, Bd } from 'cozy-ui/transpiled/react/Media'
+import Icon from 'cozy-ui/transpiled/react/Icon'
+import palette from 'cozy-ui/transpiled/react/palette'
+import { useI18n } from 'cozy-ui/transpiled/react/I18n'
+
+const WebsiteLinkCard = ({ link }) => {
+  const { t } = useI18n()
+  let url = null
+  try {
+    url = new URL(link)
+  } catch (err) {
+    logger('warn', err.message)
+    return ''
```

Comment:
Nit: `return null` is better here. No need to fix it here right away since we want a quick publication. But it'd be cool to fix it on master.